src/util/loading.cpp

```diff
@@ -0,0 +1,74 @@
+
+#include "loading.hpp"
+
+#include "world/entity/object_handler.hpp"
+
+#include <json/json.h>
+
+#include <global_context.hpp>
+
+#include <iostream>
+#include <string>
+
+namespace util {
+
+void
+load_manifest() {
+    auto manifest_folder = files::get_manifest_path();
+
+    Json::Value manifest;
+    /*for (const auto& manifest_folder_entry :
+         std::filesystem::directory_iterator(manifest_folder)) {
+        Json::Value mod_manifest;
+
+        std::filesystem::path manifest_file = manifest_folder_entry;
+
+        LOG_DEBUG(logging::file_io_logger, "Opening file {}.", manifest_file.string());
+
+        auto contents = files::open_data_file(manifest_file);
+        if (contents.has_value()) {
+            contents.value() >> manifest;
+        } else {
+            LOG_WARNING(
+                logging::file_io_logger, "Cannot open file {}.", manifest_file.string()
+            );
+        }
+
+        update_json(manifest, mod_manifest);
+    }*/
```

Comment:
Make this more permanent. Remove the comment and read from multiple manifest files.